src/components/avatar/Avatar.js

```diff
@@ -32,7 +32,7 @@ const Avatar = ({ className, name, image, preloadImage, ...rest }) => {
             <PreloadImage
                 src={ image }
                 decode={ preloadImage }
-                container={ imageContainer }
+                container={ animateOnEnter ? imageContainer : undefined }
```

Comment:
If `animateOnEnter` is true, shouldn't we simply change the `appear` prop on `<TransitionGroup>` instead?

Reply:
I can do that. But do you prefer to pass the `animateOnEnter` prop to `imageContainer` function as argument?

Reply:
Yep and it would map to appear.